iothub/service/tests/Registry/DevicesClientTests.cs

```diff
@@ -285,8 +285,8 @@ public async Task DevicesClient_SetAsync_EmptyListThrows()
         public async Task DevicesClient_SetAsync_WithOnlyIfUnchangedTrue()
         {
             // arrange
-            var goodDevice1 = new Device("123") { ConnectionState = ClientConnectionState.Connected };
-            var goodDevice2 = new Device("234") { ConnectionState = ClientConnectionState.Connected };
+            var badDevice1 = new Device("123") { ConnectionState = ClientConnectionState.Connected };
+            var badDevice2 = new Device("234") { ConnectionState = ClientConnectionState.Connected };
```

Comment:
They are not good devices because they are missing ETag properties

Reply:
You could add this comment to the test file so that it explains why these devices are "bad"!
Alternatively, 
var deviceWithoutETag1 = new Device("123") { ConnectionState = ClientConnectionState.Connected };

Reply:
Sure